### ISSUE TITLE
FXIOS-323 ⁃ [a11y] Web page bleeds through into Library screen for VoiceOver users

### ISSUE THREAD

**Issue by jcsteh** [mozilla-mobile/firefox-ios#6346]
STR:

1. Open a web page.
2. Tap Menu, then tap Your Library.
3. Turn on VoiceOver.
4. Tap the Bookmarks button at the bottom left of the screen to select it.
5. Flick left.
    - Expected: VoiceOver should report an element on the Library screen.
    - Actual: VoiceOver reports the "Backwards" button, which is one of the web page controls and is not visible on the Library screen.

Continuing to flick left will take you to content from the web page itself.

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-323)

**Comment by jcsteh** [comment]
P1 for accessibility because users will be extremely confused and might inadvertently activate controls they don't intend to activate.

**Comment by data-sync-user** [comment]
➤ Ashley Thomas commented:

Check to see is FXIOS-1490 fixes this

**Comment by data-sync-user** [comment]
➤ Ashley Thomas commented:

[~sbasca@mozilla.com] Sounds like this one has a PR up. During testing, can you check if it fixes https://jira.mozilla.com/browse/FXIOS-1490 and https://jira.mozilla.com/browse/FXIOS-322 ?

**Comment by data-sync-user** [comment]
➤ Daniela Arcese commented:

Fixed in FXIOS-1490

**Comment by SimonBasca** [comment]
Verified fix on v34.0 (4680) and main 493d272cca.
